connectors/aws/aws-dynamodb/element-templates/aws-dynamodb-outbound-connector.json

```diff
@@ -943,11 +943,11 @@
     },
     "type" : "Dropdown",
     "choices" : [ {
-      "name" : "DELETE",
```

Comment:
out of curiosity, why do we remove it?

Reply:
It was just moved a few lines down, the reason is that I made a change in the generator that preserves the order of dropdown options from the annotation.